algolia-crawler.js

```diff
@@ -29,7 +29,9 @@ new Crawler({
           // FYI selector documentation: https://github.com/cheeriojs/cheerio#selectors
           recordProps: {
             lvl0: {
-              selectors: '#site-title',
+              selectors: $(
+                'meta[name=commercetools:title-for-onsite-search]'
+              ).attr('content'),
```

Comment:
you can try out very simply in the browser dev console. The search bar at the bottom of the DOM explorer is evaluating CSS selectors.   

attr("content") looks good following the cheerio docs.  

You might want to make it a fallback logic for deployment (e.g. app kit goes live separately and would have no site name at all for a while)

Reply:
I would have waited with the actual code change in Algolia until I applied the update to all repositories. But it wouldn't hurt anyway 👍🏼